src/components/App.js

```diff
@@ -16,15 +16,36 @@ class App extends Component {
 
         const airbenderList = new AirbenderList({ characters: [] });
         main.appendChild(airbenderList.render());
-    
-        api.getCharacters()
-            .then(characters => {
-                airbenderList.update({ characters });
-            })
-            .catch(err => {
-                console.log(err);
-            });
 
+        function loadEnemies() {
```

Comment:
Does this function load Enemies? Or does it load characters based on your query. Just a small naming thing to think about.